src/commands/test.js

```diff
@@ -121,16 +121,18 @@ async function runBinary(datasource, opts) {
   args.length > 0 ? binary.run(...args) : binary.run()
 }
 
-function getPlatform(logsOpt) {
+async function getPlatform(logsOpt) {
   const type = os.type()
   const arch = os.arch()
-  const release = os.release()
   const cpuCore = os.cpus()[0]
-  const majorVersion = semver.major(release)
   const isM1 = cpuCore.model.includes("Apple M1")
+  const linuxInfo = type === 'Linux' ? await getLinuxInfo() : []
+  const linuxDestro = linuxInfo[0] || type
+  const release = linuxInfo[1] || os.release()
+  const majorVersion = parseInt(linuxInfo[1]) || semver.major(release)
```

Comment:
Not sure if it's still a good practice, but I remember that it's a "good practice" to pass the radix (10 in this case) argument to `parseInt`, because JS tries to figure it out in some cases.

But I guess that might be more of a problem in browsers? Well, I will just note here if we have issues in the future, it might be this.

Reply:
Hmmm https://stackoverflow.com/questions/44971276/why-is-it-recommended-to-provide-optional-radix-parameter-to-parseint